app/logic/events.py

```diff
@@ -14,7 +14,7 @@
 from app.models.eventTemplate import EventTemplate
 from app.models.programEvent import ProgramEvent
 from app.logic.adminLogs import createLog
-
+import datetime
```

Comment:
To avoid `datetime.datetime`, you can import like `from datetime import datetime`